client/blindbox/requests/errors.py

```diff
@@ -0,0 +1,10 @@
+class AttestationException(Exception):
```

Comment:
Minus the non documented payload part, it looks ok. but I would suggest you use it as a base exception for other errors. 
Please implement the exceptions "NotAnEnclaveError" and "InvalidEnclaveCode", using this exception as a base.

Regarding the InvalidEnclaveCode error, please refer to this to know how to implement it: 
https://github.com/mithril-security/blindai/blob/b7b7938864ae523065721dca9f546401190ec3ee/client/blindai/_dcap_attestation.py#LL73C50-L73C50

Reply:
Done